src/licensedcode/data/rules/not-a-license_432.RULE

```diff
@@ -0,0 +1 @@
+S5PC100_GPL4(0)
```

Comment:
Was this really detected?

Reply:
No, only GPL1, GPL2 and GPL3 were detected. On second thought this is really unnecessary, removing and renaming these.